inspector/src/components/actionTabs/tabs/propertyGrids/animations/animationListTree.tsx

```diff
@@ -184,18 +212,38 @@ export class AnimationListTree extends React.Component<
         const selectAnimation = () => this.props.selectAnimation(animation);
         const toggle = () => this.toggleProperty(i);
         return (
-            <li className={this.props.selected && this.props.selected.name === animation.name ? "property sub active" : "property sub"} key={i}>
-                <div className={`animation-arrow ${this.state.animationList && this.state.animationList[i].open ? "" : "flip"}`} onClick={toggle}></div>
+            <li
+                className={
+                    this.props.selected && this.props.selected.name === animation.name
+                        ? "property sub active"

```

Comment:
Spacing may be a little overdone here. Just makes it hard to read with all the separated lines.

Reply:
Correct. I added a new spacing :)